doc/guides/test/sanitycheck.rst

```diff
@@ -390,6 +394,16 @@ harness_config: <harness configuration options>
              tags: sensors
              depends_on: i2c
 
+    The following is an example yaml file with pytest harness_config options.
+
+    ::
+
+        tests:
+          test_section:
+          harness: pytest
+          harness_config:
+            handle_script: tests/.../pytest
```

Comment:
path should be relative to the test root and we should use a default if the script is not provided, also, the name 'handle_script' is misleading, it is not a script, I would call this pytest_root and use 'pytest' as a default

Reply:
fixed.  thanks.